transcendence-app/Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN apk add --no-cache dumb-init \
 WORKDIR  /service
 COPY ["package.json", "package-lock.json", "./"]
 COPY --from=development --chown=node:node /service/transcendence-app/dist ./transcendence-app/dist
+COPY --from=development --chown=node:node /service/transcendence-app/package.json ./transcendence-app/package.json
```

Comment:
The main package.json was missing and npm was only installing root monorepo deps but not backend ones

Reply:
Thank you for this fix!